docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -48,9 +48,6 @@ deployment_groups:
 
   - id: network1
     source: modules/network/vpc
-    settings:
-      network_name: hcls-cluster-net
-      subnetwork_name: primary-subnet
```

Comment:
Why was this needed?

Reply:
This was needed because if anything were to break and the was left behind, all other deployments would fail because this VPC is hardcoded.